ops/prod/_data.tf

```diff
@@ -56,6 +56,16 @@ data "azurerm_key_vault_secret" "metabase_db_uri" {
   key_vault_id = data.azurerm_key_vault.global.id
 }
 
+data "azurerm_key_vault_secret" "postgres_readonly_user" {
```

Comment:
I might change the name from "readonly" to "nophi", since we could (and probably should) have a read-only user that has full PHI access but isn't used by metabase.

Reply:
Good call, changed to nophi across all files.